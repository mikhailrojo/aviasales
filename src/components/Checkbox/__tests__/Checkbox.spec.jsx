import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import {Checkbox} from '../index';

describe('Snapshot', () => {
	it('render Checkbox', () => {
		const renderer = new ShallowRenderer();
		const component = renderer.render(
			<Checkbox
				value="0"
				checked
				onChange={jest.fn()}
				label="без пересадок"
			/>
		);
		expect(component).toMatchSnapshot();
	});
});
