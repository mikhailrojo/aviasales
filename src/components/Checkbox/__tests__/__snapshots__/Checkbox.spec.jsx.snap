// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render Checkbox 1`] = `
<styled.div>
  <label>
    <styled.span>
      <input
        checked={true}
        onChange={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
        type="checkbox"
        value="0"
      />
      <styled.div />
      без пересадок
    </styled.span>
  </label>
  <styled.span
    data-id="0"
    onClick={[Function]}
  >
    только
  </styled.span>
</styled.div>
`;
